.github/tests/images.json

```
@@ -38,6 +38,11 @@
       "query": "nodeDriverRegistrar.image",
       "filter": "^v",
       "sort-flags": []
+    },
+    {
+      "query": "selinux.image",
+      "filter": "^[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+$",
```

I thin it should maybe be:
```suggestion
      "filter": "^[0-9]\\+\\.[0-9]\\+-[0-9]\\+$",
```